CocoapodsIntegrationTest/Gemfile

```diff
@@ -2,4 +2,4 @@
 
 source "https://rubygems.org"
 
-gem 'cocoapods', "~>1.7.0.beta"
+gem 'cocoapods', "!=1.7.0.rc.1"
```

Comment:
I think, it will not fix the issue, because the file is replaced by `CocoapodsIntegrationTest/TestEnvironments/<POD_CONFIG_DIR>/Gemfile`, so we need to make changes there.

BTW, do you think we should remove this file to avoid confusion? Or maybe a comment will be enough?

Reply:
I was wondering about that. I tested locally but probably not fully. I'll put together another PR.

Thanks!

Reply:
Continued in #2944